tests/src/TestCase.php

```diff
@@ -17,6 +17,13 @@
 
 class TestCase extends \PHPUnit_Framework_TestCase
 {
+    public function setUp()
+    {
+        if ('' === ini_get('date.timezone')) {
+            date_default_timezone_set('UTC');
```

Comment:
For tests, wouldn't it be better to enforce a specific timezone for consistency?

Reply:
done